apps/edxapp/vars/all/main.yml

```diff
@@ -34,3 +34,12 @@ edxapp_nginx_image_name: "fundocker/openshift-nginx"
 edxapp_nginx_image_tag: "1.13"
 edxapp_nginx_cms_port: 8081
 edxapp_nginx_lms_port: 8071
+
+# -- celery/redis
+edxapp_celery_lms_high_priority_queue: "edx.lms.core.high"
+edxapp_celery_lms_default_priority_queue: "edx.lms.core.default"
+edxapp_celery_lms_low_priority_queue: "edx.lms.core.low"
+edxapp_celery_lms_high_mem_queue: "edx.lms.core.high_mem"
+edxapp_celery_cms_high_priority_queue: "edx.cms.core.high"
+edxapp_celery_cms_default_priority_queue: "edx.cms.core.default"
+edxapp_celery_cms_low_priority_queue: "edx.cms.core.low"
```

Comment:
In fact I realized that edX is using the same queues for lms and cms. should we still split?

Reply:
:thinking: I think we can also use the same queues. In this case, what about the workers? Should we keep two types of them?

Reply:
so in fact edx IS using different queues. It is overriden in the aws.py setting file and I was looking at the common.py setting file. This being said, we could decide to run one worker per priority level and run both the lms and the cms queues on this worker... the setting when launching a worker is "queues" plural and can take several...

I'm for running 7 containers as it is simpler...